generators/client/files-react.js

```diff
@@ -444,7 +445,8 @@ const files = {
                 'e2e/page-objects/password-page.ts',
                 'e2e/page-objects/settings-page.ts',
                 'e2e/page-objects/register-page.ts',
-                'protractor.conf.js'
+                'protractor.conf.js',
```

Comment:
these 2 files shouldnt be here as they will be added in previous block anyway

Reply:
Indeed, I'm fixing this